docs/getting-started/quick-start.md

```diff
@@ -6,9 +6,136 @@ sidebar_position: 2
 
 In this section we are providing a sample application that one can deploy and just use to play around.
 
-## K8S
+## Kubernetes
 
-_Here we provided a small sample on K8s_
+### Application
```

Comment:
We should have a mention that the service binding should be installed prior to running this quick start guide.

Reply:
Done.